src/enquiry/graphql/enquiry.graphql

```diff
@@ -39,7 +39,8 @@ input CreateEnquiryInput{
 
 type Query {
     getEnquiry(id: String!): Enquiry
-    getEnquiries: [Enquiry]
+    getCustomerEnquiries(userId: String!): [Enquiry]
+    getDestinationEnquiries(destinationId: String!): [Enquiry]
```

Comment:
getAgentEnquiries..
get all enquiries for all destinations that the agent serve